client/templates.html

```diff
@@ -91,7 +91,7 @@ <h3 data-i18n>Luokittelu</h3>
   <div class="drafts"></div>
   <label data-i18n>Luokittelu rekisteröity</label>
   <div class="classifications" data-cy="classifications"></div>
-  <label data-i18n>Ikäraja</label>
+  <label data-i18n>Ikäraja1</label>
```

Comment:
Is the added 1 at the end meant to be there?

Reply:
Left by accident. Removed it.